src/components/dataDisplays/CollapsibleRow.jsx

```diff
@@ -67,7 +68,7 @@ export default function CollapsibleRow({
               }}
             >
               {customCellRenderer ? (
-                customCellRenderer(cellValue, datum)
+                customCellRenderer(cellValue, datum, rowIndex)
```

Comment:
There are places where customBodyRender only has two arguments. E.g., CollaborationsCard.jsx and CategoryTable.jsx and SpeciesEditor.jsx... I tested SpeciesEditor, and it seems to work fine, but I don't understand how.

Reply:
So if you invoke a function with more arguments than necessary, that's not an error. 
```
function hi(a) {
  console.log("Hi ", a);
}
hi('tom', 'jim', 'harry'); // logs "Hi tom", there is no error
```
Basically that index is something you _can_ use if you want when you implement customBodyRender, but most of the time you wont need it